### PR TITLE
MICRO Stompy

### PR DESCRIPTION
**Pull request by jingxiangmo**

### CODE REVIEW

**Comment by jingxiangmo** [comment]
W&B: https://wandb.ai/64/XBot/runs/lbpe9t9r?nw=nwuserjingxiangmo

File: sim/resources/stompymicro/joints.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        return leaves
+
+    def __str__(self) -> str:
+        parts = [str(child) for child in self.children()]
+        parts_str = textwrap.indent("\n" + "\n".join(parts), "  ")
+        return f"[{self.__class__.__name__}]{parts_str}"
+
+
+
+class Hip(Node):
```
Comment (budzianowski) [inline sim/resources/stompymicro/joints.py:48]: remove - not used

File: sim/resources/stompymicro/joints.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                "lower": -1.5707963,
+                "upper": 1.5707963,
+            },
+        }
+
+    # p_gains
+    @classmethod
+    def stiffness(cls) -> Dict[str, float]:
+        return {
+            "hip pitch": 250,
```
Comment (budzianowski) [inline sim/resources/stompymicro/joints.py:151]: Update the names to:
```suggestion
            "Hip_Pitch": 250,
```
if you carefully look at you initialization you don't use any of your joints.

File: sim/envs/humanoids/stompymicro_config.py [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+        cycle_time = 0.4  # sec
+        # if true negative total rewards are clipped at zero (avoids early termination problems)
+        only_positive_rewards = True
+        # tracking reward = exp(error*sigma)
+        tracking_sigma = 5.0
+        max_contact_force = 400  # forces above this value are penalized
+
+        class scales:
+            # reference motion tracking
+            joint_pos = 1.6
```
Comment (budzianowski) [inline sim/envs/humanoids/stompymicro_config.py:166]:
```suggestion
            joint_pos = 1.6
                        feet_clearance = 1.6
            feet_contact_number = 1.2
            # gait
            feet_air_time = 1.6
            foot_slip = -0.05
            feet_distance = 0.2
            knee_distance = 0.2
            # contact
            feet_contact_forces = -0.01
            # vel tracking
            tracking_lin_vel = 1.2
            tracking_ang_vel = 1.1
            vel_mismatch_exp = 0.5  # lin_z; ang x,y
            low_speed = 0.2
            track_vel_hard = 0.5
```

for standing - comment this part ou

File: sim/resources/stompymicro/joints.py [hunk 4]
```diff
@@ -196,6 +310,12 @@ def effort(cls) -> Dict[str, float]:
             "hip roll": 80,
             "knee rotate": 80,
             "foot rotate": 80,
+            "shoulder yaw": 80,
```
Comment (budzianowski) [inline sim/resources/stompymicro/joints.py:313]: these are wrong values. your urdf joints are `  <joint name="Hip_Lift_Joint_Limit" type="revolute">` etc.

**Pull request merged**